src/types/report.js

```diff
@@ -1,7 +1,7 @@
 // @flow
 import type { Map } from 'immutable';
 
-export type ReportRow = {
+export type Report = {
```

Comment:
I don't agree on the naming as a report means set of data rows as a whole but this context defines just one singe row of data.